DbConnection/src/main/java/com/solvd/laba/db/model/Vehicle.java

```diff
@@ -0,0 +1,124 @@
+package com.solvd.laba.db.model;
+
+public class Vehicle {
+    private int id;
+    private String brand;
+    private String model;
+    private int year;
+    private String color;
+    private int mileage;
+    private String licenseNumber;
```

Comment:
Is license number a string; I'm probably wrong I'm not sure

Reply:
Yes, some license number can contains alphabets as well and also we wont be using license number for any arithmetic operations. So that's why I used string instead of long/int.